Basic-Scripts/Pattern.py

```diff
@@ -0,0 +1,17 @@
+picture = [
```

Comment:
Document your code & separate the function logic from main function. Include a main function.
It would be great if you can take dynamic input ( input from user ) [ **Optional** ]